cmd/deviceservice/add/add.go

```diff
@@ -38,19 +27,22 @@ var operState string
 var addrName string
 var file string
 
+const DeviceServicesTempl = `[{{range $ds := .}}` + update.DeviceSrviceTempl +
```

Comment:
better to rename the method s/update.DeviceSrviceTempl/update.DeviceServiceTempl

Reply:
This comment is already addressed in PR https://github.com/edgexfoundry/edgex-cli/pull/271
The current PR depends on https://github.com/edgexfoundry/edgex-cli/pull/271 and this is why the change from both PRs are visible here. 
Better approach probably was to wait the first PR to be merged.